js/src/stats.js

```diff
@@ -110,6 +130,28 @@ module.exports = (common) => {
       })
     })
 
+    it('.bw Promise Poll', (done) => {
+      if (!withGo) {
+        console.log('Not supported in js-ipfs yet')
+        return
+      }
+
+      ipfs.stats.bw({poll: true}).then((res) => {
+        expect(res).to.exist()
+
+        res.once('data', (data) => {
+          expect(data).to.have.a.property('totalIn')
+          expect(data).to.have.a.property('totalOut')
+          expect(data).to.have.a.property('rateIn')
+          expect(data).to.have.a.property('rateOut')
+          done()
+        })
+      }).catch(err => {
+        expect(err).to.not.exist()
```

Comment:
can just do `.catch(done)` instead as `done` with a error set will fail the test case

Reply:
Since you are using Promises, you can remove the `done` and just make sure that you return the promise. The way it is now you don't get any benefit from using the Promise syntax over Callbacks